tools/cross_runner/kernels/conv_nchw_fp16.cpp

```diff
@@ -205,6 +205,19 @@ _GENX_ inline void store_output_wc8_as_nchw(SurfaceIndex surface [[type("buffer_
 	}
 }
 
+_GENX_ inline void store_output_wc8_as_nhwc(SurfaceIndex surface [[type("buffer_t")]], vector_ref<DT_OUT, ACCU_REG_SIZE> grf_chunk, uint32_t output_width, uint32_t output_channels, uint32_t byte_offset)
+{
+	const uint32_t BLOCK_SC = MAX_ELEMENT_SIZE / OUTPUT_ELEMENT_SIZE;
+
+	for (int i = 0; i < BLOCK_W; i++)
```

Comment:
`#pragma unroll`   can help with performance if BLOCK_W is compile time constant

Reply:
Added